osquery/config/parsers/file_paths.cpp

```diff
@@ -81,20 +85,48 @@ Status FilePathsConfigParserPlugin::update(const std::string& source,
     data_.add("file_accesses", arr);
   }
 
-  // We know this top-level is an Object.
-  const auto& file_paths = config.at("file_paths").doc();
-  if (file_paths.IsObject()) {
-    for (const auto& category : file_paths.GetObject()) {
-      if (category.value.IsArray()) {
-        for (const auto& path : category.value.GetArray()) {
-          std::string pattern = path.GetString();
-          if (pattern.empty()) {
-            continue;
+  if (config.count("file_paths") > 0) {
+    // We know this top-level is an Object.
+    const auto& file_paths = config.at("file_paths").doc();
+    if (file_paths.IsObject()) {
+      for (const auto& category : file_paths.GetObject()) {
+        if (category.value.IsArray()) {
```

Comment:
In case of other type should we write an error to log? Because in that case it will be broken configuration.